README.md

```diff
@@ -1,2 +1,105 @@
-# optee_benchmark
-OP-TEE Latency benchmark application
+# OP-TEE Benchmark application
+## Contents
+1. [Introduction](#1-introduction)
+2. [Implementation details](#2-implementation-details)
+3. [Generating latency data](#3-generating-latency-data)
+4. [Analyzing results](#4-analyzing-results)
+
+## 1. Introduction
+
+This README contains information about examples of usage
+and implementation details of benchmark application.
+
+For a generic description of benchmark feature, its components in OP-TEE OS
+core, Linux TEE driver and libteec, please check [optee_benchmark.md].
+
+---
+
+## 2. Implementation details
+
+Benchmark application consists of two parts:
+- **Benchmark Client Application**, which is mainly used to generate latency
+  data. It should be invoked on a destination SoC.
```

Comment:
s/It should be invoked on a destination SoC./You are supposed to run this on the device/?